cron.ts

```diff
@@ -17,61 +22,75 @@ const refreshDiscordServers = async () => {
 
 export const refreshDiscordServer = async (discordServer: DiscordServer) => {
   console.log(`[Cron] Refreshing discord server ${discordServer.id}`);
+
   const discordMembers = await DiscordMemberRepository.find({
     where: {
-      discordServer,
+      discordServerId: discordServer.id,
       starknetWalletAddress: Not(IsNull()),
     },
     withDeleted: true,
   });
+
+  const discordConfigs = await DiscordServerConfigRepository.findBy({
+    discordServerId: discordServer.id,
+  });
+
   // Refreshing each member one by one. We could
   // optimize using a pool in parallel.
-  const starkyModule = modules[discordServer.starkyModuleType];
-  if (!starkyModule) {
-    console.error(
-      `Discord server ${discordServer.id} is configured with module ${discordServer.starkyModuleType} which does not exist`
-    );
-    return;
-  }
+
   for (let discordMember of discordMembers) {
-    try {
-      await refreshDiscordMember(discordServer, discordMember, starkyModule);
-    } catch (e) {
-      console.error(
-        `Could not refresh discord member ${discordMember.discordMemberId} from server ${discordServer.id}: ${e}`
-      );
+    for (let discordConfig of discordConfigs) {
+      const starkyModule = modules[discordConfig.starkyModuleType];
+      if (!starkyModule) {
+        console.error(
+          `Server configuration ${discordConfig.id} uses module ${discordConfig.starkyModuleType} which does not exist`
+        );
+        return;
```

Comment:
you may not need to return here! just need to ignore this configuration and go to the next one so use
```suggestion
        continue;
```